### PR TITLE
Updates to guidance and welsh language services guidance for review

### PR DESCRIPTION
**Pull request by romanidavies**
Updated to staging to reflect feedback from Erin and Ro's catchup with Eleri

This includes the general text added to the 'Guidance' section on the navigation bar and homepage for Mette to review.

### CODE REVIEW

**Review by accessibilitydesign — APPROVED** [review]

File: src/guidance/index.md.njk [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Guidance
+description: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```
Comment (AndyPorter) [inline src/guidance/index.md.njk:3]: can we write a description to replace this placeholder text.

e.g. guides to help design, build and operate services.

i'm not sure what a good description of this collection of guidance is.  I'd be happy to just have the fake latin replaced by something in English, and we can review for the best description later.

File: src/guidance/index.md.njk [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+layout: layout-pane.njk
+show_subnav: true
+---
+
+Here you can find guidance to help you with your design process in HM Land Registry.
+
+The guidance has been created to consider HM Land Registry's ways of working and created alongside the involved parties.
```
Comment (AndyPorter) [inline src/guidance/index.md.njk:10]: ..created **alongside** the **involved** parties.

doesn't read well to me.

could we say - created with the subject matter experts

not being a CD - i'm happy to be ignored/overruled.

File: src/guidance/welsh-guidance/index.md.njk [hunk 3]
```diff
@@ -0,0 +1,125 @@
+---
+title: Create your Welsh language service
+description: Check how accessible your content is
```
Comment (AndyPorter) [inline src/guidance/welsh-guidance/index.md.njk:3]: description looks like its been copied form another page.

could we say something like: 'guidance to help you create the Welsh version of your service'

File: src/guidance/welsh-guidance/index.md.njk [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+---
+
+## Who is this guidance for
+
+Any member of a delivery team should know:
+- when and if your service needs to provide Welsh content
+- how you can get the Welsh content
+- how we get it agreed
+- how we make sure it’s right and works for our users
+There are more things to think about when we design services that are available in both English and Welsh.
```
Comment (AndyPorter) [inline src/guidance/welsh-guidance/index.md.njk:18]: needs a blank line added to the markdown here - when i look at the [preview](https://hmlr-design-system-pr-94.herokuapp.com/guidance/welsh-guidance/) line 18 gets added to the end of the last bullet point - line  17. 
Add a blank line in-between

**Review by AndyPorter — CHANGES_REQUESTED** [review]
please have a look at my comments.

i think they need fixing before we publish.

File: src/guidance/index.md.njk (hunk 1 above)
Comment (romanidavies) [reply src/guidance/index.md.njk:3]: Replaced with: "Guidance to help you with your design process in HM Land Registry." in line with the guidance index page content, pending Mette's review.

File: src/guidance/index.md.njk (hunk 2 above)
Comment (romanidavies) [reply src/guidance/index.md.njk:10]: Amend made. Similarly, happy for CD to overrule.

File: src/guidance/welsh-guidance/index.md.njk (hunk 3 above)
Comment (romanidavies) [reply src/guidance/welsh-guidance/index.md.njk:3]: Replaced with "Guidance to help you create your Welsh language service" in keeping with language on the page referring to 'welsh language service'. Happy for CD to amend.

File: src/guidance/welsh-guidance/index.md.njk (hunk 4 above)
Comment (romanidavies) [reply src/guidance/welsh-guidance/index.md.njk:18]: Amend made

**Review by AndyPorter — APPROVED** [review]

**Review by accessibilitydesign — APPROVED** [review]

**Pull request merged**